screens/CheckoutScreen.js

```diff
@@ -170,6 +176,14 @@ export default class CheckoutScreen extends React.Component {
 
   // Adds the transaction to the user's account and updates their points.
   confirmTransaction = async transactionInfo => {
+    // Clerk Training: if storeId is "Clerk Training"'s ID
+    // do not create the transaction or update points
+    // Navigate to Confirmation Screen with a pre-filed transaction ID
+    const storeId = await AsyncStorage.getItem('storeId');
+    if (JSON.parse(await AsyncStorage.getItem('trainingMode'))) {
+      this.props.navigation.navigate('Confirmation', createFakeTransaction(transactionInfo));
```

Comment:
Code nit: name your prop and use it correspondingly in `ConfirmationScreen` for consistency with transactionId prop and clarity.